features/steps/tabs.py

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+
+"""
+Step implementations for paragraph-related features
+"""
+
+from behave import given, then, when
+
+from docx import Document
+from docx.text.tabstop import TabStop
+from docx.shared import Inches
+
+from helpers import test_docx
+
+
+# given ===================================================
+@given('a tab_stops having {count} tab stops')
+def given_a_tab_stops_having_count_tab_stops(context, count):
+    count_int = {
+        '0': 0,
+        '3': 1
+    }[count]
+    document = Document(test_docx('tab-stops'))
+    paragraph_format = document.paragraphs[count_int].paragraph_format
+    context.tab_stops = paragraph_format.tab_stops
+
+
+# when ====================================================
+
+@when('I add a tab_stop')
+def when_i_add_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops.add_tab()
+    tab.position = Inches(1.75)
+
+
+@when('I delete a tab_stop')
+def when_i_delete_a_tab_stop(context):
+    tab_stops = context.tab_stops
+    tab = tab_stops[1]
+    tab.delete()
```

Comment:
Can you help me understand your reasoning?

The behavior of TabStops.remove(tab_stop) is to remove tab_stop as a member of the TabStops sequence. So it seems natural to me that the call is on TabStops. After tab_stop is removed, it would no longer have a parent and so a .remove() call on it would be invalid.

So I'm not seeing what we would get by having both, and there would be additional complexity to account for if we did. Also, I'd like to reuse base Python method names whenever possible to minimize surprising behavior.